probe/data.go

```diff
@@ -79,6 +85,7 @@ func GetResultData(name string) *Result {
 
 // CleanData removes the items in resultData not in []Prober
 func CleanData(p []Prober) {
+	mutex.Lock()
```

Comment:
This will cause deadlock, because `GetResultData` will use Rlock within the Wlock. Just extract a function `_getResultData` to get the result without lock. Then `GetResultData` and `CleanData` could use it on their own.

Reply:
revert the fix.